File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex2.c
#include <stdio.h>

main()
{
    //vars
    float a,b,c,res;

    printf("\t\t\tMultiplicacao entre 3 numeros\n\n");

    //Entradas
    printf("Insira o 1o numero: ");
    scanf("%f",&a);

    printf("Insira o 2o numero: ");
    scanf("%f",&b);

    printf("Insira o 3o numero: ");
    scanf("%f",&c);

    //Processamento
    res = a * b * c;

    //Saída
    printf("%.2f * %.2f * %.2f = %.2f", a, b, c, res);


}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex2.java
//Importando a classe Scanner
import java.util.Scanner;

public class Ex2{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	double n1, n2, n3, res;
	
	System.out.println("Multiplicação entre três números\n\n");
	
	//Entrada
	System.out.print("Insira o 1° número: ");
	n1 = input.nextDouble();
	
	System.out.print("Insira o 2° número: ");
	n2 = input.nextDouble();
		
	System.out.print("Insira o 3° número: ");
	n3 = input.nextDouble();
	
	//Processamento
	res = n1 * n2 * n3;
	
	//Saída
	System.out.printf("%.2f * %.2f * %.2f = %.2f", n1, n2, n3, res);
		
	}
}